docs/tutorials/uniq-factories/creating-uniq-factories/how-to-validate-uniq-metadata-using-schema-validator-toolkit.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "How to validate uniq metadata using Ultra Toolkit's Schema Validator"
+order: 3
+outline: [0, 4]
+---
+
+# How to validate uniq metadata using Ultra Toolkit's Schema Validator
+
+Schema validator is a feature of the Ultra Toolkit that helps you to validate your uniq metadata files.
+
+## Prerequisites
+
+-   You must have created uniq metadata files. See [How to create Uniq Metadata](./how-to-create-uniq-metadata.md) for more information.
+
+## Goal
+
+The goal of this tutorial is to help validate your metadata files using Ultra Toolkit's schema validator.
+
+## Accessing the Toolkit
+
+To access the Ultra Toolkit, please visit: https://toolkit.ultra.io
+
+## Schema Validator
+
+Once on the toolkit home page, click on the `Schema Validator` from the sidebar to open the schema validator page.
+
+![](./images/schema-validator-default.png)
+
+Schema validator allows you to validate the metadata files for both; uniq factories and uniqs.
+
+1. To use the validator, simply select the metadata file of your choice.
+2. Specify the file type; either `Factory` (for uniq factories), or `Uniq` (for uniq tokens).
```

Comment:
Maybe mention that `Factory` is for `factory.json` only. And the `Uniq` is for anything else (Uniqs + Default Uniq)